### ISSUE TITLE
No 'draw_lines_with_colors' for lines with gradients

### ISSUE THREAD

**Issue by Evolocity1** [pvcraven/arcade#329]
## Bug Report
this is just me running the exact example code from the documentation online.

### Actual behavior: throwing error in console saying 

> rect = arcade.create_rectangle_filled_with_colors(points, colors)
> AttributeError: module 'arcade' has no attribute 'create_rectangle_filled_with_colors'

### Expected behavior:
to produce a gradient shape

### Steps to reproduce/example code:

```
import arcade

SCREEN_WIDTH = 800
SCREEN_HEIGHT = 600
SCREEN_TITLE = "Gradients Example"


class MyGame(arcade.Window):
    """
    Main application class.
    """

    def __init__(self, width, height, title):
        super().__init__(width, height, title)

        arcade.set_background_color(arcade.color.BLACK)

        self.shapes = arcade.ShapeElementList()
        color1 = (215, 214, 165)
        color2 = (219, 166, 123)
        points = (0, 0), (SCREEN_WIDTH, 0), (SCREEN_WIDTH, SCREEN_HEIGHT), (0, SCREEN_HEIGHT)
        colors = (color1, color1, color2, color2)
        rect = arcade.create_rectangle_filled_with_colors(points, colors)
        self.shapes.append(rect)

        color1 = (165, 92, 85, 255)
        color2 = (165, 92, 85, 0)
        points = (100, 100), (SCREEN_WIDTH - 100, 100), (SCREEN_WIDTH - 100, 300), (100, 300)
        colors = (color2, color1, color1, color2)
        rect = arcade.create_rectangle_filled_with_colors(points, colors)
        self.shapes.append(rect)

        color1 = (7, 67, 88)
        color2 = (69, 137, 133)
        points = (100, 400), (SCREEN_WIDTH - 100, 400), (SCREEN_WIDTH - 100, 500), (100, 500)
        colors = (color2, color1, color2, color1)
        shape = arcade.draw_lines_with_colors(points, colors, line_width=5)
        self.shapes.append(shape)

        color1 = (215, 214, 165)
        color2 = (219, 166, 123)
        color3 = (165, 92, 85)
        points = (SCREEN_WIDTH // 2, 500), (SCREEN_WIDTH // 2 - 100, 400), (SCREEN_WIDTH // 2 + 100, 400)
        colors = (color1, color2, color3)
        shape = arcade.create_triangles_filled_with_colors(points, colors)
        self.shapes.append(shape)

        color1 = (69, 137, 133, 127)
        color2 = (7, 67, 88, 127)
        shape = arcade.create_ellipse_filled_with_colors(SCREEN_WIDTH // 2, 350, 50, 50,
                                                         inside_color=color1, outside_color=color2)
        self.shapes.append(shape)

    def on_draw(self):

        arcade.start_render()
        self.shapes.draw()

def main():

    MyGame(SCREEN_WIDTH, SCREEN_HEIGHT, SCREEN_TITLE)
    arcade.run()


if __name__ == "__main__":
    main()```

**Comment by pvcraven** [comment]
Which version is this on? `examples/gradients.py` seems to be working on 2.0.1.

**Comment by pvcraven** [comment]
`draw_lines_with_colors` seems to be missing.

**Comment by Evolocity1** [comment]
Is there a tutorial for that

**Comment by pvcraven** [comment]
No, just API docs and this example: http://arcade.academy/examples/gradients.html#gradients

**Comment by Evolocity1** [comment]
I get this error when I copy the example code from those API docs

**Comment by Evolocity1** [comment]
I'm using pycharm and have arcade downloaded through that

**Comment by Evolocity1** [comment]
It just seems to be the rectangle that isnt working.

**Comment by pvcraven** [comment]
Fix checked into dev version

**Comment by pvcraven** [comment]
Fixed in 2.0.8 released 2019-04-27.
